$OpenBSD: patch-wwwroot_inc_remote_php,v 1.1 2013/04/17 14:25:51 sthen Exp $

fix 500 server error if protocol was not specified in terminal_settings
subroutine; 75d85b67 upstream

--- wwwroot/inc/remote.php.orig	Wed Apr 17 15:19:40 2013
+++ wwwroot/inc/remote.php	Wed Apr 17 15:19:50 2013
@@ -323,7 +323,7 @@ function makeGatewayParams ($object_id, $tolerate_remo
 			$params_from_settings[] = $settings['hostname'];
 			break;
 		default:
-			throw RTGatewayError ("Invalid terminal protocol '${settings['protocol']}' specified");
+			throw new RTGatewayError ("Invalid terminal protocol '${settings['protocol']}' specified");
 	}
 
 	foreach ($params_from_settings as $param_name => $setting_name)
